framework_tvb/tvb/interfaces/web/templates/jinja2/flow/type2component/number2component.html

```diff
@@ -41,12 +41,6 @@
                 <tr> <td colspan="2" id="{{ containerDivId }}_rangeLabelsTd"> </td> </tr>
             </table>
 
-            <p class="field-adapters">
```

Comment:
I think all templates under `type2component` folder are no longer in use, right?
If that is the case, they should be removed

Reply:
From my part this task is done. Please let me know if there is anything else I can delete.

Reply:
the rebase seems to be done in the wrong direction. Unfortunately now the changes list is too difficult to read. Could you cleanup history?

Reply:
Done.